src/screens/ks/KSDashboard.js

```diff
@@ -1,8 +1,6 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import DashboardSection from './components/DashboardSection';
-import '../../styles/KSDashboard.css';
```

Comment:
Are we not using KSDashboard.css anymore?

Reply:
nice catch, adding that back in now